test-app/src/main/java/org/readium/r2/testapp/shared/views/Group.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Readium Foundation. All rights reserved.
+ * Use of this source code is governed by the BSD-style license
+ * available in the top-level LICENSE file of the project.
+ */
+
+package org.readium.r2.testapp.shared.views
+
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+
+/**
+ * Sets the emphasis (alpha) of a group of [Composable] views.
+ */
+@Composable
+fun Group(lowEmphasis: Boolean = false, enabled: Boolean = true, content: @Composable () -> Unit) {
```

Comment:
This is not really necessary, but I find the [recommended approach for de-emphasizing elements](https://developer.android.com/jetpack/compose/compositionlocal) really heavy.